docs/software/exportformat.md

```diff
@@ -157,6 +157,28 @@ As not every state has the same field requirements, not every method of completi
 | Embed URL             | For submissions done inside an embed, the URL of the embed                      |
 | Session ID            | The ID of a user's session. For tracking users across multiple tools or visits. |
 
+## Upcoming Tool Export Format
```

Comment:
Looks like we're adding `Lookup Type` to the export as well (https://github.com/vote/turnout/blob/main/app/reporting/runner.py) so we should add that here and explain what it means.